corporate/models.py

```diff
@@ -353,6 +353,11 @@ class CustomerPlan(AbstractCustomerPlan):
     # and not every time when cron run.
     invoice_overdue_email_sent = models.BooleanField(default=False)
 
+    # Flag to track if an email has been sent to Zulip team to
+    # review the pricing, 60 days before the end date. Helps to send
+    # an email only once and not every time when cron run.
+    reminder_to_review_plan_email_sent = models.BooleanField(default=False)
```

Comment:
While naming this field, I didn't included the term "fixed-price-plan" because, I think, we can reuse this field to track reminder email sent for Enterprise plans.

Reply:
Yeah, it seems like a reasonable generic mechanism.